cmd/integrations.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"github.com/urfave/cli"
+
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/integrations"
+)
+
+var (
+	IntegrationsListCommand = cli.Command{
+		Name:     "integrations",
+		Category: "Integrations",
+		Usage:    "List your external integrations",
+		Description: `List all the external integrations associated with your account:
+
+	$ scalingo integrations
+
+	# See also commands 'integrations-create', 'integrations-destroy', 'integrations-import-keys'`,
+
+		Action: func(c *cli.Context) {
+			err := integrations.List()
+			if err != nil {
+				errorQuit(err)
+			}
+		},
+		BashComplete: func(c *cli.Context) {
+			_ = autocomplete.CmdFlagsAutoComplete(c, "integrations")
+		},
+	}
+
+	IntegrationsCreateCommand = cli.Command{
+		Name:     "integrations-create",
+		Category: "Integrations",
+		Flags: []cli.Flag{appFlag,
+			cli.StringFlag{Name: "url", Usage: "URL of the integration", Value: "<url>", EnvVar: ""},
+			cli.StringFlag{Name: "token", Usage: "Token of the integration", Value: "<token>", EnvVar: ""},
+		},
+		Usage: "Create a link between external integration and your account",
+		Description: `Create a link between external integration and your account:
+
+	For github.com and gitlab.com:
+	$ scalingo integrations-create github
+					  OR
+	$ scalingo integrations-create gitlab
+
+	For github enterprise and gitlab self hosted instances:
+	$ scalingo integrations-create integration-type --url integration-url --token integration-token
+
+	Examples:
+	$ scalingo integrations-create github-enterprise --url https://ghe.example.com --token ...
+	$ scalingo integrations-create gitlab-self-hosted --url https://gitlab.example.com --token ...
+
+	# See also commands 'integrations', 'integrations-destroy', 'integrations-import-keys'`,
+
+		Action: func(c *cli.Context) {
+			var err error
+			if len(c.Args()) >= 1 && len(c.Args()) <= 5 {
+				url := c.String("url")
+				if url == "<url>" {
+					url = ""
+				}
+
+				token := c.String("token")
+				if token == "<token>" {
+					token = ""
+				}
+
+				err = integrations.Create(c.Args()[0], url, token)
+			} else {
+				_ = cli.ShowCommandHelp(c, "integrations-create")
+			}
+
+			if err != nil {
+				errorQuit(err)
+			}
+		},
+		BashComplete: func(c *cli.Context) {
+			_ = autocomplete.CmdFlagsAutoComplete(c, "integrations-create")
+		},
+	}
+
+	IntegrationsDestroyCommand = cli.Command{
+		Name:     "integrations-destroy",
+		Category: "Integrations",
+		Usage:    "Destroy a link between external integration and your account",
+		Description: `Destroy a link between external integration and your account:
+
```

Comment:
I Don't like the expression "external integration". Isn't "SCM integration" more descriptive?